src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/thread/WallThreadComponent.java

```diff
@@ -0,0 +1,51 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.messagewall.thread;
+
+import com.wikia.webdriver.common.contentpatterns.URLsContent;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.wikia.webdriver.common.contentpatterns.URLsContent'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)